sidebars.js

```diff
@@ -167,6 +167,7 @@ const sidebars = {
   'spl-docs/top-command',
   'spl-docs/transaction-command',
   'spl-docs/where-command',
+  'spl-docs/extract-command',
```

Comment:
This should be inserted in alphabetic order like the rest. Fixed by https://github.com/siglens/siglens-docs/pull/121